packages/eslint-plugin-dialtone/lib/rules/recommend-typography-style.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileoverview Utilities to set Font family, Font weight, Font size, and Line height separately are discouraged in favor of composed typography utilities
+ * @author Nina Repetto
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'suggestion', // `problem`, `suggestion`, or `layout`
+    docs: {
+      description: "Utilities to set Font family, Font weight, Font size, and Line height separately are discouraged in favor of composed typography utilities",
+      recommended: false,
+      url: 'https://github.com/dialpad/eslint-plugin-dialtone/blob/main/docs/rules/recommend-typography-style.md', // URL to the documentation page for this rule
+    },
+    fixable: null, // Or `code` or `whitespace`
+    schema: [], // Add a schema if the rule has options
+    messages: {
+      recommendTypographyStyle: `Utilities to set Font family, Font weight, Font size, and Line height separately are
+      discouraged in favor of composed typography utilities. Checkout the available classes here:
+      https://dialtone.dialpad.com/design/typography/#api`,
```

Comment:
Let's add "There can be cases where using these utilities is intentional and valid, in which case you can ignore this lint warning."